frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Button as MuiButton, makeStyles } from "@material-ui/core";
+
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        margin: theme.spacing(0.5)
+    },
+    label: {
+        textTransform: 'none'
+    }
+}))
+
+export default function Button(props) {
+
+    const { text, size, color, variant, onClick, ...other } = props
+    const classes = useStyles();
+
+    return (
+        <MuiButton
+            variant={variant || "contained"}
+            size={size || "large"}
+            color={color || "primary"}
+            onClick={onClick}
```

Comment:
The alternative values here can just be the defaults. If you define some type here:
`const ButtonType = {
    variant?: variantType,
    size?: sizeType
    ....
}`.

Then you can set the defaults.
The ? next to the name marks the argument as optional.
Check out the Pick function for grabbing exact types from MUI. I think it would end up looking something like this:
`const ButtonType = Pick<MuiButtonType, "variant" | "size" | "color" | "onClick"> & {
...other stuff
}`

Reply:
I just removed the button component overall.